dev/tools/svg2dart/lib/svg2dart.dart

```diff
@@ -0,0 +1,204 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+import 'dart:math';
+
+import 'package:collection/collection.dart';
+import 'package:xml/xml.dart' as xml show parse;
+import 'package:xml/xml.dart' hide parse;
+
+/// Interprets an SVG file.
```

Comment:
what we discussed -- this should set expectations lower

Reply:
done